tests/test_meshtastic_utils.py

```
@@ -5146,6 +5159,64 @@ def test_connect_meshtastic_ble_interface_creation_timeout(
             ]
             self.assertEqual(len(warning_calls), MAX_TIMEOUT_RETRIES_INFINITE)
 
+    @patch("mmrelay.meshtastic_utils.logger")
+    @patch("mmrelay.meshtastic_utils.time.sleep")
+    def test_connect_meshtastic_ble_signature_unavailable_uses_compatibility_mode(
+        self, _mock_sleep, mock_logger
```

Ruff (0.15.9)

[warning] Fixture `_mock_sleep` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)